example/lib/widgets/controls.dart

```diff
@@ -154,20 +154,35 @@ class _ControlsWidgetState extends State<ControlsWidget> {
     }
     if (WebRTC.platformIsAndroid) {
       // Android specific
-      try {
+      requestBackgroundPermission([bool retryTwice = false]) async {
         // Required for android screenshare.
-        const androidConfig = FlutterBackgroundAndroidConfig(
-          notificationTitle: 'Screen Sharing',
-          notificationText: 'LiveKit Example is sharing the screen.',
-          notificationImportance: AndroidNotificationImportance.Default,
-          notificationIcon:
-              AndroidResource(name: 'livekit_ic_launcher', defType: 'mipmap'),
-        );
-        await FlutterBackground.initialize(androidConfig: androidConfig);
-        await FlutterBackground.enableBackgroundExecution();
-      } catch (e) {
-        print('could not publish video: $e');
+        try {
+          bool hasPermissions = await FlutterBackground.hasPermissions;
+          if (!retryTwice) {
+            const androidConfig = FlutterBackgroundAndroidConfig(
+              notificationTitle: 'Screen Sharing',
+              notificationText: 'LiveKit Example is sharing the screen.',
+              notificationImportance: AndroidNotificationImportance.Default,
+              notificationIcon: AndroidResource(
+                  name: 'livekit_ic_launcher', defType: 'mipmap'),
+            );
+            hasPermissions = await FlutterBackground.initialize(
+                androidConfig: androidConfig);
+          }
+          if (hasPermissions &&
+              !FlutterBackground.isBackgroundExecutionEnabled) {
+            await FlutterBackground.enableBackgroundExecution();
+          }
+        } catch (e) {
+          if (!retryTwice) {
```

Comment:
The original issue is here https://github.com/livekit/client-sdk-flutter/issues/195#issuecomment-1296542050, probably due to some delay after the FlutterBackground plugin calls FlutterBackground.initialize for the first time, an exception will be thrown for the first time, and it will succeed when `FlutterBackground.enableBackgroundExecution();` is executed after waiting for 1 second, and this phenomenon Appears only when installing the app for the first time and enabling screen sharing on Android.

I think it is a bug of FlutterBackground, but I created this temporary fix to solve the crash when starting screen sharing for the first time, that is to call it again after receiving the first exception

Reply:
This is an experimental fix, I think the problem is that the https://pub.dev/packages/flutter_background plugin failed to execute the first time, rather than an issue with livekit-sdk-flutter itself.

Reply:
Got it. is there any ways we could put this in the README to let developers know? I think every user will run into this bug, and it makes the SDK look unreliable.